dbt_copilot_helper/COMMANDS.md

```diff
@@ -710,6 +714,60 @@ copilot-helper domain assign --app <app> --env <env> --svc <svc>
 - `--help <boolean>` _Defaults to False._
   - Show this message and exit.
 
+# copilot-helper domain cdn
+
+[↩ Parent](#copilot-helper-domain)
+
+    Assigns a CDN domain name to application loadbalancer.
+
+## Usage
+
+```
+copilot-helper domain cdn --project-profile <project_profile> --env <env> 
+                          --app <app> --svc <svc> [--delete] 
```

Comment:
I would prefer the delete to be a separate command.

Reply:
yes, will move delete out

Reply:
new cdn-delete command